scripts/wheels/linux/setup.py

```diff
@@ -0,0 +1,34 @@
+import setuptools
+import os
+
+#
+# with open("README.md", "r") as fh:
+#     long_description = fh.read()
+
+class BinaryDistribution(setuptools.Distribution):
+    def has_ext_modules(foo):
+        return True
+
+
+class EmptyListWithLength(list):
+    def __len__(self):
+        return 1
+
+setuptools.setup(
+    name="KratosMultiphysicsWheelTest",
+    version="0.1",
```

Comment:
I would use the official version (7.0+hash(last master commit))

Reply:
done